### ISSUE TITLE
Show comments in Edit mode

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/social-paper#37]
There was a conscious choice not to show comments in Edit mode, I think because we (I) thought it would be too complicated to show them both at once. It turns out that it probably isn't all that complicated, so maybe we should do it. I'm not sure who built this particular piece - was it you @r-a-y ?

If there are any technical hurdles then we definitely should not do this for v1.

**Comment by mkgold** [comment]
It might be worth thinking about a show/hide comments toggle in edit mode. I'd hate to lose the clean writing interface but Erin is right that some people may want to see comments as they are writing.

**Comment by r-a-y** [comment]
>  I think because we (I) thought it would be too complicated to show them both at once. It turns out that it probably isn't all that complicated, so maybe we should do it. I'm not sure who built this particular piece - was it you @r-a-y ?

This is tied to the comment auto-repositioning issue.  At the moment, we do not show comments because repositioning comments requires a page refresh.

**Comment by boonebgorges** [comment]
Ah right. This should definitely be tabled then.

**Comment by sheesh** [comment]
Agreed that it should be tabled for now if complicated. Thanks for the
thoughts.

erin rose glass | digital fellow | cuny grad center | nyc
www.erinroseglass.com | @erinroseglass
